File: rso.py
def solve(Y: int, N: str) -> str:
    if len(N) > 50:
        return False
    elif "Cal " in N or "California " in N or "Berkeley " in n:
        if Y < 2010:
            return True
        else:
            return False
    else:
        return True

def main():
    T = int(input())
    for _ in range(T):
        Y = int(input())
        N = input()
        print('VALID' if solve(Y, N) else 'INVALID')

if __name__ == '__main__':
    main()

File: rso.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int t;
    cin >> t;

    for (int i = 0; i < t; i++) {
        int y;
        cin >> y;
        for (int j = 0; j < 50; j++) {
            
        }
        string n;
        cin >> n;

        if (y < 2010) {
            cout << "INVALID" << endl;
        } else if (n.size() > 50) {
            cout << "INVALID" << endl;
        } else if (n.find("Cal ") || n.find("Berkeley ") || n.find("California ")) {
            cout << "INVALID" << endl;
        } else {
            cout << "VALID" << endl;
        }
    }
}